src/dex/jarvis-v6/config.ts

```diff
@@ -0,0 +1,308 @@
+import { DexParams } from './types';
+import { DexConfigMap, AdapterMappings } from '../../types';
+import { Network, SwapSide } from '../../constants';
+import { Interface } from '@ethersproject/abi';
+import ChainlinkAccessControlledOffchainAggregatorABI from '../../abi/jarvis/ChainlinkAccessControlledOffchainAggregator.json';
+import SynthereumPriceFeedABI from '../../abi/jarvis/SynthereumPriceFeed.json';
+
+export const JarvisV6Config: DexConfigMap<DexParams> = {
+  JarvisV6: {
+    [Network.POLYGON]: {
+      priceFeed: {
+        interface: new Interface(SynthereumPriceFeedABI),
+        address: '0x12F513D977B47D1d155bC5ED4d295c1B10D6D027',
+      },
+      pools: [
+        {
+          address: '0x65a7b4Ff684C2d08c115D55a4B089bf4E92F5003',
+          priceFeedPair: 'EURUSD',
+          syntheticToken: {
+            address: '0x4e3Decbb3645551B8A19f0eA1678079FCB33fB4c',
+            decimals: 18,
+            symbol: 'jEUR',
+          },
+          collateralToken: {
+            address: '0x2791bca1f2de4661ed88a30c99a7a9449aa84174',
+            decimals: 6,
+            symbol: 'USDC',
+          },
+          chainLink: {
+            interface: new Interface(
```

Comment:
it'll be less noisy to maintain only addresses here and instantiate  chainLink once only like [here](https://github.com/paraswap/paraswap-dex-lib/blob/8adeb6af1af93e7be7dfc8ba53d8f3f06cdadb50/src/dex/jarvis-v6/jarvis-v6.ts#L50).
Minor though